lib/config/env/all.js

```diff
@@ -6,12 +6,17 @@ var rootPath = path.normalize(__dirname + '/../../..');
 
 module.exports = {
   root: rootPath,
-  port: process.env.PORT || process.env.NODEJS_PORT || 3000,
-  hostname: process.env.NODEJS_IP || undefined,
+  port: process.env.PORT || 3000,
+  hostname: undefined,
```

Comment:
This was key in getting GAE Flex working. If I put anything in here it would cause forwarding from the GAE proxy to Express to fail and give `503` errors.